src/commonMain/kotlin/com/mattprecious/stacker/remote/NoRemote.kt

```diff
@@ -19,7 +19,7 @@ class NoRemote : Remote {
 	override fun openOrRetargetPullRequest(
 		branchName: String,
 		targetName: String,
-		prInfo: () -> Remote.PrInfo
+		prInfo: () -> Remote.PrInfo,
```

Comment:
Oh man what happened here. I guess I ran spotless on this branch and not the other one? That's annoying. Need to add a CI check.